src/client/lib/format-number.js

```diff
@@ -0,0 +1,23 @@
+const toNumber = (value) => {
```

Comment:
we'll have to refactor these functions a bit as they should no longer contain the `this` keyword.